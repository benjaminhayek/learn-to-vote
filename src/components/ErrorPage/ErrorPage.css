.error-title {
    text-align: center;
    color: red;
}