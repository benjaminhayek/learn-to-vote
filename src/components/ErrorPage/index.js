import React from 'react';

const ErrorPage = () => {
    return(
        <div>
            <h1>That is not a valid address</h1>
        </div>
    )
}

export default ErrorPage;