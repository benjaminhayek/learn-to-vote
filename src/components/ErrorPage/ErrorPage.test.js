import React from 'react';
import ErrorPage from './index';
import { shallow } from 'enzyme';

describe('ERROR PAGE', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<ErrorPage />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})