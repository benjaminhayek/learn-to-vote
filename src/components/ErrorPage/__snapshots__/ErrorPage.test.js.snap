// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ERROR PAGE should match the snapshot 1`] = `
<div>
  <h1
    className="error-title"
  >
    Sorry that is not a valid address :(
  </h1>
</div>
`;
